### ISSUE TITLE
Accessibility: Missing alternative texts for images

### ISSUE THREAD

**Issue by vvvt** [corona-warn-app/cwa-website#94]
## Where to find the issue
1. Decorative images in the section _Wie funktioniert die App?_ or _How does the app work?_ on the main page
2. GitHub Icon in the header and the footer
3. Card images for news articles on `/news`

## Describe the issue
1. The decorative images on the main page have no alternative text.
2. The GitHub icon functions as a link but lacks an alternative text to describe its destination 
3. The images, used in the news cards have no alternative text.

## Suggested change
1. Add an empty alternative text `alt=""` to declare the image as purely decorative, [as recommended by the WCAG](https://www.w3.org/WAI/tutorials/images/decorative/#image-used-as-part-of-page-design).
2. Add an alternative text, describing the GitHub repository as the links destination, for example `alt="GitHub Repository"`
3. Add an alternative text to the `<img/>` tag. Depending on, whether these images are deemed purely decorative or functional, the alt text has to be empty `alt=""` or a description of the iamge content.

**Comment by ingomc** [comment]
Already included in other PR
Duplicate of #65

**Comment by ayrustogaru** [comment]
Can I work on this?

**Comment by corneliusroemer** [comment]
@ingomc and @ayrustogaru 
Before you get started on work, it may be helpful getting the maintainers to confirm they are interested in your help. Otherwise your work may be wasted.
This issue is relevant: #99 
Maintainers, your feedback is requested :) @SebastianWolf-SAP @tkowark @jakobmoellersap

**Comment by mojoaxel** [comment]
Some of this is fixed in #146
